Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Cancel.hs

```diff
@@ -180,6 +185,7 @@ cancel booking mRide req cancellationSource = do
         merchant = merchant,
         cancelStatus = show Enums.CONFIRM_CANCEL,
         vehicleVariant = DVeh.castServiceTierToVariant booking.vehicleServiceTierType,
+        cancellationReason = Just $ SCR.getCancellationReasonCode req.reasonCode,
```

Comment:
we can add isValueAddNP check here itself, debugging will be easier in this way. Otherwise passing Just here and then conditional makes it misleading.

Reply:
Done